### PR TITLE
Fb marq

### PR DESCRIPTION
**Pull request by alex-tiago**
Front-end updates to Marquee

### CODE REVIEW

File: apps/slides/admin.py [hunk 1]
```diff
@@ -17,25 +17,25 @@ class SlideAdmin(admin.ModelAdmin):
...
+		'text',
+		'slide_scheme',
+		'color_bg',
+		'color_font',
+		'font',
 		'font_size',
 		'display_start',
 		'display_end',
-		'location',
+		#'location',
```
Comment (rjucf) [inline apps/slides/admin.py]: The code that has been commented out and is not used should be removed. Remove code throughout that has been commented out.

**Review by rjucf — APPROVED** [review]

**Review by vinhtvu2 — APPROVED** [review]

File: templates/base.djt [hunk 2]
```diff
@@ -43,7 +43,7 @@
 
 		<!-- FOOTER -->
 		<div id="footer">
-			<p><a id="ucf_logo" href="http://www.ucf.edu/">UCF Stands For Opportunity</a></p>
+			<p><a id="ucf_logo" href="http://www.ucf.edu/">University of Central Florida</a></p>
```
Comment (cadie) [inline templates/base.djt]: We can now use https for this URL:
`https://www.ucf.edu/`

**Review by cadie — APPROVED** [review]

File: static/style/screen.css [hunk 3]
```diff
@@ -8,16 +8,16 @@
 /******************************************************************************\
     General Styles
 \******************************************************************************/
-body { font: 80% "Trebuchet MS", sans-serif; text-align:left; }
+body { font: 90% "Gotham SSm A","Gotham SSm B",sans-serif; text-align:left; }
```
Comment (cjg89) [inline static/style/screen.css]: Gotham font usage on this project won't work without a Cloud.typography CSS key.  You'll need to talk to Roger about incorporating one of our CSS keys to make this work, or consider just using Helvetica + Georgia as fallbacks.

File: apps/slides/models.py [hunk 4]
```diff
@@ -17,54 +17,53 @@ class Slide(models.Model):
...
 	font		  = models.CharField(default='Gotham', max_length='255', choices=fonts_tuple())
 	font_size     = models.IntegerField("Font Size", default=16, choices=FONTSIZES)
 	color_bg	  = models.CharField("Background Color", max_length="6", default="000000")
 	color_font	  = models.CharField("Font Color", max_length="6", default="EBB700")
 	display_start = models.DateField()
 	display_end	  = models.DateField()
-	event_start	  = models.DateTimeField(null=True)
-	event_end	  = models.DateTimeField(null=True)
-	location	  = models.CharField("Event Location", max_length='255', blank=True)
 	org			  = models.CharField("Name of UCF Organization / Unit", max_length=255)
```
Comment (cjg89) [inline apps/slides/models.py:27]: We'll need to get migrations set up and included in this PR before it can be merged in.

**Review by cjg89 — CHANGES_REQUESTED** [review]

File: apps/slides/admin.py (hunk 1 above)
Comment (alex-tiago) [reply apps/slides/admin.py]: removed commented lines

File: templates/base.djt (hunk 2 above)
Comment (alex-tiago) [reply templates/base.djt]: replaced http with https

File: static/style/screen.css (hunk 3 above)
Comment (alex-tiago) [reply static/style/screen.css]: Jim added UCF Sans Serif Alt

File: apps/slides/models.py (hunk 4 above)
Comment (alex-tiago) [reply apps/slides/models.py:27]: Migrations done by Jim

**Review by rjucf — APPROVED** [review]

**Review by jmbarne3 — APPROVED** [review]

**Review by cjg89 — APPROVED** [review]

**Review by cadie — APPROVED** [review]

**Pull request merged**